avm/res/app/container-app/main.bicep

```diff
@@ -337,13 +337,6 @@ resource containerApp_diagnosticSettings 'Microsoft.Insights/diagnosticSettings@
           timeGrain: null
         }
       ]
-      logs: [
```

Comment:
Please then also update the type to onlyMetrics. It's available in the avm-common-types like the current type